src/js/_enqueues/admin/application-passwords.js

```diff
@@ -57,7 +57,7 @@
 			$newAppPassButton.prop( 'disabled', false );
 
 			$newAppPassForm.after( tmplNewAppPass( {
-				name: name,
+				name: response.name,
 				password: response.password
```

Comment:
This change displays the sanitized name instead of the given raw name. Why? To avoid confusion with the name that is saved and shown in the table.

An example name: `<script>alert('Hello')</script> hello` will be cleaned to be `hello`. 

Before:
<img width="715" alt="51941-name-notice-before" src="https://user-images.githubusercontent.com/7284611/105579809-60ee6300-5d4e-11eb-896a-a6b2c0a41677.png">

After:
<img width="651" alt="51941-name-notice-after" src="https://user-images.githubusercontent.com/7284611/105579811-68157100-5d4e-11eb-9d5d-1fc90f669c5d.png">

Reply:
@TimothyBJacobs @mrxkon What do you think of this change? It bothered me that the raw `name` was being displayed in the "Your new password for" message while the sanitized name is the actual name.

Reply:
This looks like a good change to me.